src/ipc/capnp/echo.capnp

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2021 The Bitcoin Core developers
```

Comment:
b2abe3f062c69663ae97bed239aa0a118e6a081c

Apologies for the remedial question, but this code *isn't* autogenerated, right?

Reply:
> For an interface to be shared across processses, it needs two definitions:
> 
> * An **API definition** declaring how the interface is called. Examples: [node.h](https://github.com/ryanofsky/bitcoin/blob/ipc-export/src/interfaces/node.h), [wallet.h](https://github.com/ryanofsky/bitcoin/blob/ipc-export/src/interfaces/wallet.h), [echo.h](https://github.com/ryanofsky/bitcoin/blob/ipc-export/src/interfaces/echo.h), [init.h](https://github.com/ryanofsky/bitcoin/blob/ipc-export/src/interfaces/init.h)
> * A **data definition** declaring how interface calls get sent across the wire. Examples: [node.capnp](https://github.com/ryanofsky/bitcoin/blob/ipc-export/src/ipc/capnp/node.capnp), [wallet.capnp](https://github.com/ryanofsky/bitcoin/blob/ipc-export/src/ipc/capnp/wallet.capnp), [echo.capnp](https://github.com/ryanofsky/bitcoin/blob/ipc-export/src/ipc/capnp/echo.capnp), [init.capnp](https://github.com/ryanofsky/bitcoin/blob/ipc-export/src/ipc/capnp/init.capnp)
> 
> Since API and data definitions contain some of the same information, they have to be kept in sync manually when methods or parameters change. This is designed to be _straightforward_ and _safe_. _Straightforward_ because there is no need to write manual serialization code or use awkward intermediate types like [`UniValue`](https://github.com/bitcoin/bitcoin/blob/master/src/univalue/include/univalue.h) instead of native types. _Safe_ because if there are any inconsistencies between API and data definitions (even minor ones like using a narrow int data type for a wider int API input), there are errors at build time instead of errors or bugs at runtime.

This was helpful to me, a version of this info could go into `doc/multiprocess.md` IMO.